src/style.css

```diff
@@ -311,14 +311,6 @@ article footer a:hover {
 	content: attr(data-default);
 }
 
-/* Admin Bar */
```

Comment:
I had the same thought and first had d3d32ff7a2eee8a9, but then reverted in 975eb568c6c01 because I think the Spine calculates `top` automatically and adjusts. I almost added a test page with a wp-admin bar to the Spine repo so that we could see it in action, but then got lazy. I'll do that to be sure.

Reply:
And you are correct! There was different behavior depending on whether the nav/content combo was short/long or long/short. I've addressed that in the Spine and the CSS here is still needed. 👍

Reply:
Sheesh!